src/command_modules/azure-cli-backup/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+1.1.0
++++++
+* Added new command 'az backup protection isenabled-for-vm'. This command can be used to check if a VM is backed up by any vault in the subscription.
+* Enabled --ids for vault_name and resource_group parameters.
```

Comment:
This should call out which commands now support `--ids`.  Note that `list` commands should *not* support `--ids`. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-the-ids-parameter

New guidance was added for removing `--ids` from list commands.